.eslintrc.js

```diff
@@ -71,13 +71,13 @@ module.exports = {
     'no-undef-init': 'error',
     'no-unused-expressions': 'error',
     'no-unused-labels': 'error',
-    'no-use-before-define': ['warn', {functions: false, classes: true}],
+    'no-use-before-define': 'off', // TS takes care of this one...
```

Comment:
Typescript ensures this is safe. Do we have a reason to have a lint rule? This lint rule triggers on completely safe patterns like:

```ts
const getFoo() => {
  return getBar()
}

const getBar() => {
  return 'bar';
}